cms/djangoapps/contentstore/views/course.py

```diff
@@ -89,7 +88,8 @@
 from util.milestones_helpers import (
     set_prerequisite_courses,
     is_valid_course_key,
-    is_prerequisite_courses_enabled
+    is_prerequisite_courses_enabled,
```

Comment:
These booleans really do not belong in milestones_helpers.py, but it's an additional refactoring job to move them into their own common modules/libraries, so we won't tackle it as part of this pull request.